en-us/server-setup.md

```diff
@@ -18,33 +18,33 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
+  * If you're using a plugin that uses system messages for chat (e.g. the ones below), the report button will be disabled,
+  * If you're using vanilla-like chat, vanilla users may see a red bar on the left of the message and ![red markings](https://i.ibb.co/ftRMqHL/exclamation.png) on the right on FO users' messages. FO users can report vanilla users, but vanilla users cannot report FO users,
+  * If you're using the [No Chat Reports](https://www.curseforge.com/minecraft/mc-mods/no-chat-reports) mod, FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) and the report button will be disabled.
+* If your server is based on 1.19 or up and chat signatures are enforced, FO users will have their messages signed as well. FO users will see ![red ⚠️](https://i.ibb.co/tzd8CvB/red.png) on the bottom right corner of the chat box.
+* If you're using Realms, everyone is monitored by Mojang and has chat messages signed. FO users will see ![red ⚠️ with two exclamation marks](https://i.ibb.co/WcVggrL/chat-status-icons-extended.png) on the bottom right corner of the chat box.
 
 #### How to protect your users
 
 - **All 1.18- backend servers**: no changes necessary, chat reporting doesn't exist.
 - **All 1.19+ backend servers**: set `enforce-secure-profile` to `false` in _server.properties_
-  - This doesn't disable chat reporting by itself, but allows users to join without requiring them to sign their messages, to protect their privacy.
+  - This doesn't disable chat reporting by itself, but allows users to join without requiring them to sign their messages, to protect their privacy,
   - If no other measures are taken alongside this, _vanilla clients_ will see a warning toast in the top right corner and they will still sign the messages. That means anyone can report them, but they cannot report FO users. To avoid those problems, use one of the plugins/mods below.
 - **Velocity**: set `force-key-authentication` to `false` in _velocity.toml_
   - Same comments apply as for "all servers" above.
 - **BungeeCord/Waterfall**: set `enforce_secure_profile` to `false` in _config.yml_
   - Same comments apply as for "all servers" above.
-- **Paper/Pufferfish/Purpur/Spigot**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin - no config required
-- **Fabric/Quilt/Forge**: install _one_ of the following plugins:
-  - [NoChatReport](https://modrinth.com/mod/no-chat-report) - compatible with most chat mods/vanilla chat, no config required 
-  - [No Chat Reports](https://www.curseforge.com/minecraft/mc-mods/no-chat-reports) - same mod as in FO; FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat
+- **Spigot/Paper/Pufferfish/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin and ensure `rewrite-chat`, `claim-secure-chat-enforced` and `send-prevents-chat-reports-to-client` are all set to `true` inside the FreedomChat's config folder for best effects.
```

Comment:
```suggestion
- **Spigot/Paper/Pufferfish/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin and ensure `rewrite-chat`, `claim-secure-chat-enforced` and `send-prevents-chat-reports-to-client` are all set to `true` inside the FreedomChat's config folder. FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat.
```

Reply:
Also, `the FreedomChat's config folder`?